pkg/azurediskplugin/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.20.6
+FROM alpine:3.18.12
```

Comment:
/hold
`alpine:3.18.12` is already EOL now, that's the reason we want to upgrade to 3.20